src/browser/components/JobDescription.react.js

```diff
@@ -0,0 +1,36 @@
+import Component from 'react-pure-render/component';
+import React from 'react';
+
+if (process.env.IS_BROWSER) {
+  require('./JobDescription.styl');
+}
+
+export default function JobDescription(props) {
+  const data = props.data;
+  return (
+    <div className="content job">
+      <div className="thin">
+        <h1>
+          {data.name}
+        </h1>
+        <p className="screen-desc">
+          {data.who}
+        </p>
+        <h3>What is your mission in Scalac?</h3>
+        <ul>
+          {
+            data.what && data.what.map(item => {
```

Comment:
`what` to znaczy?

Reply:
tutaj mozesz bez `return` w jednej linii

Reply:
The same applies to other names